config/routes.rb

```diff
@@ -2,7 +2,7 @@
   namespace :api, defaults: {format: :json} do
     namespace :v1 do
       namespace :invoices do
-        get 'find', to 'find#show'
+        get 'find', to: 'find#show'
```

Comment:
do we need a '/find' for a relative path?

Reply:
yes. will fix now.